QEfficient/cloud/infer.py

```diff
@@ -60,8 +60,10 @@ def main(
     device_group: List[int] = [
         0,
     ],
+    skip_stats : bool = False,
```

Comment:
@vbaddi @hupreti 
Can we please make a new function called `export_and_compile` inside infer API that returns `qpc_path` and has the logic of not regenerating the `qpc/ONNX` if they already exist, 

This way:
 - `infer.py::main` can use this function
 - as well as app,
 - as well as tests that we are planning
 
We can meet if you guys want me to explain this.  Please let me know if you guys have a better solution.

Reply:
restored cli interface, we need to have a single interface which will perform all things for user so it is not just compile and execute

Reply:
Can we move the `latency_kv_stats` call in the main function and remove it from `infer_api`
get `qpc path`, `model_hf_path` from `infer_api` prepare tokenizer using `model_hf_path` and pass `tokenizer` and `qpc_path`  to `latency_kv_stats` inside main function.

I agree that we need single interface to perform all, that will be main function in the file.

And that way, I think you don't have to use any extra parameter. You can also rename `infer_api` to `export_and_compile` then.